src/Symfony/Component/JsonStreamer/Mapping/Read/DateTimeTypePropertyMetadataLoader.php

```diff
@@ -38,7 +38,7 @@ public function load(string $className, array $options = [], array $context = []
             $type = $metadata->getType();
 
             if ($type instanceof ObjectType && is_a($type->getClassName(), \DateTimeInterface::class, true)) {
-                if (\DateTime::class === $type->getClassName()) {
+                if (\DateTime::class === $type->getClassName() || is_a($type->getClassName(), \DateTime::class, true)) {
```

Comment:
I think that was the suggestion :)

```suggestion
                if (is_a($type->getClassName(), \DateTime::class, true)) {
```

Reply:
oh, of course 😄